using_images/xpaas_images/sso.adoc

```diff
@@ -0,0 +1,166 @@
+. Click *Create* to deploy the application template.
+
+=== Deployment Process
+Once deployed, three pods will be created, two pods for the SSO web servers and one for the database. A readiness check prevents the web server pods from starting until the database pod is running. After the SSO web server pods have started, they can be accessed at their custom configured hostnames, or at the default hostnames:
+
+* _http://sso-{project-name}.{hostname}/auth_ - for the web server, and
+* _https://secure-sso-{project-name}.{hostname}/auth_ - for the encrypted web server.
+
+The default login username/password credentials are _admin_/_admin_.
+
+== Quickstart Example: Using the Red Hat Single Sign-On xPaaS Image with the SSO-enabled JBoss EAP xPaaS Image
+This example uses the OpenShift web console to deploy Red Hat Single Sign-On xPaaS backed by a PostgreSQL database. Once deployed, an SSO realm, role, and user will be created to be used when configuring the SSO-enabled JBoss EAP xPaaS Image deployment. Once successfully deployed, the SSO user can then be used to authenticate and access JBoss EAP. 
+
+=== Deploy the SSO Application Template
+
+. Log in to the OpenShift web console with a user that holds _cluster-admin_ privileges and select the {project-name} project space.
+. Click *Add to project* to list all of the default image streams and templates.
+. Use the *Filter by keyword* search bar to limit the list to those that match _sso_. You may need to click *See all* to show the desired application template.
+. Click the *_sso70-postgresql_* application template to list all of the deployment parameters. These parameters will be left as default for this example. 
+. Click *Create* to deploy the application template and start pod deployment. This may take a couple of minutes.
+
+=== Create SSO Credentials
+Log in to the encrypted SSO web server at _https://secure-sso-{project-name}.{hostname}/auth_ using the default _admin_/_admin_ username and password.
+
+* *Create a Realm*
+
+. Create a new realm by hovering your cursor over the realm namespace (default is *Master*) at the top of the sidebar and click the *Add Realm* button. 
+. Enter a realm name and click *Create*.
+
+* *Copy the Public Key*
+In the newly created realm, click the *Keys* tab and copy the public key that has been generated. This will be needed to deploy the SSO-enabled JBoss EAP image.
+
+* *Create a Role*
+
+. Click *Roles* in the *Configure* sidebar to list the roles for this realm. As this is a new realm, there should only be the default _offline_access_ role. Click *Add Role*.
+. Enter a role name and optional description and click *Save*.
+
+[NOTE]
+This role correseponds to the security role (app-jee, app-profile-jee, or app-profile-jee-saml) as defined in the *web.xml* of the application. The example application uses a jee role with the role name of _user_.
```

Comment:
NOTE boxes should have 4 ='s above and below the para: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions